Update docker-entry binary to fix ashmem driver cannot compile issue.

lwan89 opened the pull request:
Tracked-On: OAM-88747
Signed-off-by: Wang, Liang <liang.wang@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/68

Done by "Luo, LilianX" <lilianx.luo@intel.com>

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/69

Done by "Luo, LilianX" <lilianx.luo@intel.com>

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckApprovals Some checks are failed. See the step results.html

More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/72

sysopenci reviewed (commented):
Failure when executing preintegration builder

FAILURE:CheckBug Bad comments/Bugs

More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/73

sysopenci reviewed (commented):
PR integrated into 'preintegration' builder



More details, logs, and binaries: /absp/builders/celadon_pmr0_cic-preintegration/builds/74

Done by "Luo, LilianX" <lilianx.luo@intel.com>

Commit:
Fall back to older binderfs driver

binderfs driver ported from Ubuntu 5.0 cause high CPU usage of
audio server. Fall back to older version for now.

Tracked-On: OAM-88787
